src/validateRegisterForm.test.js

```diff
@@ -3,28 +3,145 @@
 describe(`Function 'validateRegisterForm':`, () => {
   const validateRegisterForm = require('./validateRegisterForm');
 
-  it(`should be declared`, () => {
-    expect(validateRegisterForm).toBeInstanceOf(Function);
-  });
+  describe(`Generic tests`, () => {
+    it(`should be declared`, () => {
+      expect(validateRegisterForm).toBeInstanceOf(Function);
+    });
 
-  it(`should return object`, () => {
-    expect(typeof validateRegisterForm('test@mail.com', 'P@ssword1!'))
-      .toBe('object');
+    it(`should return object`, () => {
+      expect(typeof validateRegisterForm('test@mail.com', 'P@ssword1!'))
+        .toBe('object');
+    });
   });
 
-  it(`should return success message for the valid input`, () => {
-    const isValid = validateRegisterForm('test@mail.com', 'P@ssword1!');
+  describe(`Email and password validation`, () => {
+    it(`should return success message for the valid input (Aa-Zz)`, () => {
```

Comment:
(Aa-Zz) - email or password, or both? Please make test names as descriptive as you can